### ISSUE TITLE
Upgrade eksctl in otomi-tools 

### ISSUE THREAD

**Issue by j-zimnowoda** [redkubes/otomi-core#197]
The eksctl 0.24.0 does not support creation of 1.18 Kubernetes cluster.

**Comment by Morriz** [comment]
Its already deploying 1.18 clusters tho. Check the values

**Comment by j-zimnowoda** [comment]
Are you sure. I was able to deploy it but only after upgrading my eksctl.
I see it was upgraded in tools/Dockerfile but apparently this image has never been released.

I am trying to build it now but I am getting errors:
```
W: GPG error: https://packages.microsoft.com/repos/azure-cli bionic InRelease: At least one invalid signature was encountered.
E: The repository 'https://packages.microsoft.com/repos/azure-cli bionic InRelease' is not signed.
W: GPG error: https://download.docker.com/linux/ubuntu bionic InRelease: At least one invalid signature was encountered.
E: The repository 'https://download.docker.com/linux/ubuntu bionic InRelease' is not signed.
W: GPG error: http://ppa.launchpad.net/rmescandon/yq/ubuntu bionic InRelease: At least one invalid signature was encountered.
E: The repository 'http://ppa.launchpad.net/rmescandon/yq/ubuntu bionic InRelease' is not signed.
W: GPG error: http://security.ubuntu.com/ubuntu bionic-security InRelease: At least one invalid signature was encountered.
E: The repository 'http://security.ubuntu.com/ubuntu bionic-security InRelease' is not signed.
W: GPG error: http://archive.ubuntu.com/ubuntu bionic InRelease: At least one invalid signature was encountered.
E: The repository 'http://archive.ubuntu.com/ubuntu bionic InRelease' is not signed.
W: GPG error: http://archive.ubuntu.com/ubuntu bionic-updates InRelease: At least one invalid signature was encountered.
E: The repository 'http://archive.ubuntu.com/ubuntu bionic-updates InRelease' is not signed.
W: GPG error: http://archive.ubuntu.com/ubuntu bionic-backports InRelease: At least one invalid signature was encountered.
E: The repository 'http://archive.ubuntu.com/ubuntu bionic-backports InRelease' is not signed.
```
Please be aware that I do not build from cache.

**Comment by Morriz** [comment]
I see it was released manually (it was already in 1.4.7 but latest is 1.4.8), which is what we should prevent from now on. We should create a separate pipeline/workflow for this that just listens to changes to the `tools/Dockerfile`, and we could split the cloud tools into `tools/Dockerfile_cloud` at the same time
